setup.py

```diff
@@ -64,6 +64,7 @@
         "presidio-anonymizer==2.2.351",
         "usaddress==0.5.10",
         "nemo_toolkit[nlp]>=1.23.0",
+        "lxml[html_clean]",
```

Comment:
Do you want to reference the upstream issue incase changes are made to justest directly?